templates/components/molecules/media-block/media-block.twig

```diff
@@ -1,4 +1,4 @@
 <div{{ attributes }} class="decanter-media_block {{ classes }}">
-  {% include "@decanter/components/atoms/media-block-img/media-block-img.twig" with { "classes": '' } %}
-  {% include "@decanter/components/atoms/media-block-body/media-block-body.twig" with { "classes": '' } %}
+  {% include "../../atoms/media-block-img/media-block-img.twig" with { "classes": '' } %}
```

Comment:
Where did the namespace go? This is important to keep if we plan to use the templates in other systems.

Reply:
@sherakama : I was looking for that namespace!  I don't know where it went either, perhaps we can look into that together.

Reply:
Yeha, this is something I can help with. It probably has something to do with either this not getting called: https://github.com/SU-SWS/decanter/blob/master/kss-config.json or a syntax difference in versions.

Reply:
I noticed that this syntax stopped working after upgrading kss. I didn't see any doc in the change log explaining it, tho.